examples/metrics/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/machinefi/w3bstream-wasm-golang-sdk/log"
+	"github.com/machinefi/w3bstream-wasm-golang-sdk/metrics"
+)
+
+func main() {}
+
+//export start
+func _start(rid uint32) int32 {
+	log.Log(fmt.Sprintf("start rid: %d", rid))
+
+	randomName := []string{
+		"MuchSports",
+		"Righthaja",
+		"BroadcastAut",
+		"HunterSkunky",
+		"SparklingUnique",
+	}
+	metricsJSON := `{"user": "` + randomName[rand.Intn(len(randomName))] + `", "score": ` + fmt.Sprintf("%d", rand.Intn(100)) + `}`
```

Comment:
`fmt.Sprintf("{\"user\": ....%s...%d...", randomName, randomNumber)` would be clearer